### ISSUE TITLE
[Bug]: Snapshots fail with null values using 11.0.0-rc.2

### ISSUE THREAD

**Issue by brandonroberts** [thymikee/jest-preset-angular#1148]
### Version

11.0.0-rc.2

### Steps to reproduce

Clone repo: 

```sh
git clone git@github.com:brandonroberts/jest-preset-angular.git jest-preset-angular-snapshots-v13
cd jest-preset-angular-snapshots-v13
```

Checkout repro branch

```sh
git checkout snapshot-null-values
```
Go to v13 example

```sh
cd examples/example-app-v13
```

Install deps

```sh
yarn
```

Run tests

```sh
npm run test
```

Note failing test. If the value with the `null` property is removed, the test passes successfully

### Expected behavior

The snapshot accounts for properties with null values

### Actual behavior

The test fails

```
    TypeError: Cannot convert undefined or null to object
        at hasOwnProperty (<anonymous>)
```

### Additional context

Reverting this change causes the test to pass successfully

https://github.com/thymikee/jest-preset-angular/pull/1113/files#diff-7c8b690a46bc43d11b61313c0ec840cdfd8871231374a18047389991a2a8ec19L116

### Environment

```shell
System:
    OS: macOS 11.5.2
    CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
  Binaries:
    Node: 14.16.0 - ~/.nvm/versions/node/v14.16.0/bin/node
    Yarn: 1.22.0 - ~/.yarn/bin/yarn
    npm: 6.14.11 - ~/.nvm/versions/node/v14.16.0/bin/npm
  npmPackages:
    jest: ^27.3.1 => 27.3.1
```

**Comment by ahnpnl** [comment]
fixed in **11.0.0-rc.3**

**Comment by brandonroberts** [comment]
Thanks @ahnpnl! Works correctly now!
